var/spack/repos/builtin/packages/graphviz/package.py

```diff
@@ -29,33 +29,64 @@
 
 class Graphviz(AutotoolsPackage):
     """Graph Visualization Software"""
-    homepage = "http://www.graphviz.org"
-    url      = "http://www.graphviz.org/pub/graphviz/stable/SOURCES/graphviz-2.38.0.tar.gz"
+    homepage = 'http://www.graphviz.org'
+    url      = 'http://www.graphviz.org/pub/graphviz/stable/SOURCES/graphviz-2.38.0.tar.gz'
 
     version('2.38.0', '5b6a829b2ac94efcd5fa3c223ed6d3ae')
 
-    # By default disable optional Perl language support to prevent build issues
-    # related to missing Perl packages. If spack begins support for Perl in the
-    # future, this package can be updated to depend_on('perl') and the
-    # ncecessary devel packages.
-    variant(
-        'perl', default=False,
-        description='Enable if you need the optional Perl language bindings.')
+    # We try to leave language bindings enabled if they don't cause build issues
+    # or add dependencies.
+    variant('swig', default=False,
+        description='Enable for optional swig language bindings.')
+    variant('sharp', default=False,
+        description='Enable for optional sharp language bindings.')
+    variant('go', default=False,
+        description='Enable for optional go language bindings.')
+    variant('guile', default=False,
+        description='Enable for optional guile language bindings.')
+    variant('io', default=False,
+        description='Enable for optional io language bindings.')
+    variant('java', default=False,  # Spack has no Java support
+        description='Enable for optional java language bindings.')
+    variant('lua', default=False,
+        description='Enable for optional lua language bindings.')
+    variant('ocaml', default=False,
+        description='Enable for optional ocaml language bindings.')
+    variant('perl', default=False,    # Spack has no Perl support
+        description='Enable for optional perl language bindings.')
+    variant('php', default=False,
+        description='Enable for optional php language bindings.')
+    variant('python', default=False,    # Build issues with Python 2/3
+        description='Enable for optional python language bindings.')
+    variant('r', default=False,
+        description='Enable for optional r language bindings.')
+    variant('ruby', default=False,
+        description='Enable for optional ruby language bindings.')
+    variant('tcl', default=False,
+        description='Enable for optional tcl language bindings.')
```

Comment:
How many of these variants actually build? It seems like a lot of them are missing dependencies. Can you try enabling all of them and see if you can build it?